packages/mobile/src/screens/chat-screen/ChatListItem.tsx

```diff
@@ -79,19 +106,35 @@ export const ChatListItem = ({ chatId }: { chatId: string }) => {
         {otherUser ? (
           <>
             <View style={styles.contentRoot}>
-              <ProfilePicture
-                profile={otherUser}
-                style={styles.profilePicture}
-              />
               <View style={styles.userContainer}>
-                <View style={styles.userNameContainer}>
-                  <Text style={styles.userName}>{otherUser.name}</Text>
-                  <UserBadges user={otherUser} hideName />
+                <ProfilePicture
+                  profile={otherUser}
+                  style={styles.profilePicture}
+                />
+                <View style={styles.userTextContainer}>
+                  <View style={styles.userNameContainer}>
+                    <Text style={styles.userName}>{otherUser.name}</Text>
+                    <UserBadges user={otherUser} hideName />
+                  </View>
+                  <Text style={styles.handle}>@{otherUser.handle}</Text>
                 </View>
-                <Text style={styles.handle}>@{otherUser.handle}</Text>
               </View>
+              {chat?.unread_message_count && chat?.unread_message_count > 0 ? (
```

Comment:
do we need the `&` here?

Reply:
think better to have it cuz `unread_message_count` is undefined sometimes.